.github/workflows/make-plugin-linux.yml

```diff
@@ -13,18 +13,95 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Build plugin docker image
-      run: docker-compose build prereqs protoc-plugin
-    - name: Copy binary from Docker image
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1  
+      with:
+        install: true
+
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+      with:
+        image: tonistiigi/binfmt:latest
+
+    # Temporary fix for https://github.com/bazelbuild/bazel/issues/11379
+    - name: Reset QEMU 
+      run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Docker meta prereqs
+      id: docker_meta_prereqs
+      uses: docker/metadata-action@v3
+      with:
+        images: grpcweb/prereqs
+        tags: type=semver,pattern={{version}},value=${{ github.event.inputs.version_number }}
+        flavor: latest=true
+
+    - name: Build and push Docker image prereqs
+      id: docker_build_prereqs
+      uses: docker/build-push-action@v2
```

Comment:
curious where are the images being pushed to?

Reply:
This is also a workaround for the sequential build problem. The images will be pushed with the tags specified on line 52, i.e. `grpcweb/prereqs:<version>`.

Reply:
Pushing is now no longer required, the `protoc_plugin` build successfully waits for the `prereq` build to complete.

Reply:
Aha i see thanks for addressing this! :)